corehq/apps/export/models/new.py

```diff
@@ -1124,6 +1158,40 @@ def _generate_schema_from_xform(xform, case_updates, langs, app_id, app_version)
 
         return schema
 
+    @staticmethod
+    def _get_stock_items_from_question(question, app_id, app_version, repeats):
+        """
+        Creates a list of items from a stock type question
+        """
+        items = []
+
+        # Strips the last value in the path
+        # E.G. /data/balance/entry --> /data/balance
+        parent_path = question['value'][:question['value'].rfind('/')]
+        question_id = question['stock_type_attributes']['type']
```

Comment:
this will fail hard if the form doesn't declare a type attribute, right? again, probably fine but we could add an assertion or something to make it clearer